src/components/ElvaYBPractice2/MainComponent.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react'
+import { Slider, Switch } from 'antd'
+
+import IconCheck from '../../assets/ElvaYongbin/images/icon-check.svg'
+import SliderIcon from '../../../../assets/ElvaYongbin/images/icon-slider.svg'
+
+import {
+  MainComponent,
+  PageViewContainer,
+  PageViewPriceContainer,
+  PriceMonthContainer,
+  PriceContainer,
+  MonthContainer,
+  Slidercontainer,
+  StyleSlider,
+  ToggleContainer,
+  ToggleContainerText,
+  DiscountButton2,
+  DiscountButton1,
+  MainBottomComponent,
+  BottomLeftContainer,
+  BottomLeftText1,
+  BottomLeftText2,
+  BottomLeftText3,
+  TrialButtonContainer,
+  CheckIcon,
+  StartTrialButton,
+} from './styles/MainComponent.styled'
+const marks = {
+  0: '',
+  25: '',
+  50: '',
+  75: '',
+  100: '',
+}
+
+const MainComponentContainer = () => {
+  const [views, setViews] = useState(50)
+  const [toggle, setToggle] = useState(false)
+  return (
+    <MainComponent>
+      <PageViewPriceContainer>
```

Comment:
Consider abstracting Page View Container where u pass in the boolean value of toggle and value of view as props. You can add useState hooks inside that component to change the content being displayed.